pact-tests/Pact/Core/Test/StaticErrorTests.hs

```diff
@@ -573,7 +576,7 @@ executionTests =
         (defun foo:string ())
         )
     |])
-  , ("enforce_ns_define_namespace", isExecutionError _DefineNamespaceError, [text|
+  , ("enforce_ns_define_namespace", isExecutionError _NativeExecutionError, [text|
```

Comment:
It might seem like it did, but `NativeExecutionError` includes the native name where it occurred, along with the error message. 

The constructor this replaced was `DefineNamespaceError Text` which would just say something like `Error defining namespace` as opposed to `Error during native execution define-namespace`.

I didn't see a reason for `DefineNamespaceError` when it's simply native + static error message string. Most error constructors that have their own special case, are because we want to hold data to render in them later. In the case of most static error strings, they became `NativeExecutionError`

Reply:
That sounds very reasonable. Agree with this!